foreman.toml

```diff
@@ -1,4 +1,5 @@
 # This file is used as a way to make sure that setup-foreman is functioning.
 
 [tools]
-rojo = { source = "Roblox/rojo", version = "0.6.0-alpha.1" }
\ No newline at end of file
+rojo = { source = "Roblox/rojo", version = "0.6.0-alpha.1" }
+selene = { source = "Kampfkarren/selene", version = "0.15.0" }
```

Comment:
if I don't have this here, the test with working directory fails: https://github.com/Roblox/setup-foreman/runs/7569087575?check_suite_focus=true#step:8:21

@amatosov-rbx is that the expected behavior?